webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -106,23 +83,20 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
         );
         return clusterNamespacePairs.namespaceBindings;
       } catch (e) {
-        if (!isNonK8RMError(e)) {
-          handleError(e, {
-            level: ErrorLevel.Error,
-            publicMessage: 'Failed to fetch list of workspace namespace bindings.',
-            silent: false,
-            type: ErrorType.Server,
-          });
-        }
+        handleError(e, {
+          level: ErrorLevel.Error,
+          publicMessage: 'Failed to fetch list of workspace namespace bindings.',
+          silent: true,
```

Comment:
let's try to avoid silent errors and instead just don't call the API in the first place if there's an "expected" error

Reply:
fixed